### PR TITLE
Add python doc for dynamic_lstm

### LINKED ISSUE #7641: Add python doc for dynamic_lstm

### PR DESCRIPTION
**Pull request by kuke**
Resolve #7641

### CODE REVIEW

File: python/paddle/v2/fluid/layers/nn.py [hunk 1]
```diff
@@ -229,6 +229,102 @@ def dynamic_lstm(input,
...
+    Returns:
+        tuple: The hidden state, and cell state of LSTM. The shape of both \
+        is (T x D), and lod is the same with the `input`.
+
+    Examples:
+        .. code-block:: python
+
+            hidden_dim = 512
+            forward_proj = fluid.layers.fc(input=input_seq, size=hidden_dim * 4,
+                                           act='tanh', bias_attr=True)
```
[the remark this answers is not in the document]
Comment (kuke) [reply python/paddle/v2/fluid/layers/nn.py]: @pkuyym I noticed that param `act` is not used in `fc`: https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/fluid/layers/nn.py#L63
So why didn't add activation in `fc`?

[the remark this answers is not in the document]
Comment (pkuyym) [reply python/paddle/v2/fluid/layers/nn.py]: > Note that these :math:`W_{xi}x_{t}, W_{xf}x_{t}, W_{xc}x_{t}, W_{xo}x_{t}`
> operations on the input :math:`x_{t}` are NOT included in this operator.
> Users can choose to use fully-connect layer before LSTM layer.

[the remark this answers is not in the document]
Comment (kuke) [reply python/paddle/v2/fluid/layers/nn.py]: Modified the example, set `act` and `bias_attr` to `None`

**Pull request merged**